crates/storage/codecs/derive/src/compact/mod.rs

```diff
@@ -164,7 +164,7 @@ pub fn get_bit_size(ftype: &str) -> u8 {
         "TxType" => 2,
         "u64" | "BlockNumber" | "TxNumber" | "ChainId" | "NumTransactions" => 4,
         "u128" => 5,
-        "U256" => 6,
+        "Value" | "U256" => 6,
```

Comment:
ah right, fixed.

Reply:
Hmm unfortunately this fix makes the main_codec proptests fail.

Reply:
because cfg(test) doesnt get picked up on `get_bit_size`. there's some rust issues (i dont recall exactly), where the test definition does not get propagated to all crates.

so you'd probably have to add some test features.

atm, it's encoding/decoding with U256... but `get_bit_size` is expecting it to be u128

Reply:
```rust
#[cfg(any(test, feature = "test-utils"))]
```

usually we have tackled it with this

Reply:
I added a "value_256" feature which I activate in dev-dependencies when the reth-codec dependency is included. This seems to make the tests pass, though I'm not 100% sure I've done this correctly or idiomatically!

Reply:
Having a non-test soecific here should be useful for op-reth which needs to activate the 256 bit support. Maybe I should have the test-utils feature activate value_256?